ckan/tests/lib/search/test_query.py

```diff
@@ -353,7 +353,7 @@ def test_name_multiple_results(self):
         factories.Dataset(name="second-record")
         factories.Dataset(name="third-dataset")
         result = search.query_for(model.Package).run({"q": u"record"})
-        assert set(result["results"]) == {"first-record", "second-record"}
+        assert set(result["results"]) == {"second-record", "first-record"}
```

Comment:
Please don't include unrelated changes. If there is an issue with this test it will be fixed in a separate PR

Reply:
> Please don't include unrelated changes. If there is an issue with this test it will be fixed in a separate PR

@amercader , I have revert back the unrelated changes. Thanks for the review  : )